spanner/src/update_database.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright 2023 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/main/spanner/README.md
+ */
+
+namespace Google\Cloud\Samples\Spanner;
+
+// [START spanner_update_database]
+use Google\Cloud\Spanner\SpannerClient;
+
+/**
+ * Updates the drop protection for a database.
+ * Example:
+ * ```
+ * update_database($instanceId, $databaseId, $enableDropProtection);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ * @param string $databaseId The Spanner database ID.
+ * @param bool $enableDropProtection The leader instance configuration used by default.
+ */
+function update_database(string $instanceId, string $databaseId, bool $enableDropProtection = true): void
```

Comment:
```suggestion
function update_database(string $instanceId, string $databaseId): void
```
Shall we avoid passing in enableDropProtection as an argument here for consistency across clients? Also, it'll be difficult to add more fields later on. We can reset the field in the test if required, for cleanup later on.

Reply:
I am not in favour of removing the optional argument. it will be really unclear.
Pls consider again.

Reply:
Can you please explain what will be unclear and why? IMO, the focus is on updating a database and not on drop database protection. PM team aligns on this need.

Reply:
Looking at published samples, seems like extra param will be an odd one out. So removing it.